modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureDM.testcase

```diff
@@ -38,118 +44,110 @@ definition {
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		DMDocument.deleteCP(dmDocumentTitle = "Document_1");
+			DMDocument.deleteCP(dmDocumentTitle = "Document_1");
 
-		DMDocument.deleteCP(dmDocumentTitle = "Document_2");
+			DMDocument.deleteCP(dmDocumentTitle = "Document_2");
+		}
 	}
 
-	@description = "This is a test for LPS-134093. Verify if warning message appear after try insert more than 10 documents to collect the digital signature"
+	@description = "LPS-134093. Verify if warning message appear after try insert more than 10 documents to collect the digital signature"
 	@priority = 5
 	test CanViewWarningMessageForMoreThan10Documents {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
-
-		DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.jpg,Document_2.pdf,Document_3.png,Document_4.jpg,Document_5.txt,Commerce_Black.jpg,Commerce_Blue.jpg,Commerce_White.jpg,Spaceport.jpg,analytics-cloud-hero-image.png,marketplace_icon.png");
-
-		PortletEntry.selectAll();
+		task ("Given that user Test creates multiple documents") {
+			DMDocument.addMultipleCP(dmDocumentFileList = "Document_1.jpg,Document_2.pdf,Document_3.png,Document_4.jpg,Document_5.txt,Commerce_Black.jpg,Commerce_Blue.jpg,Commerce_White.jpg,Spaceport.jpg,analytics-cloud-hero-image.png,marketplace_icon.png");
+		}
 
-		Click.javaScriptClick(
-			key_fieldName = "_com_liferay_document_library_web_portlet_DLAdminPortlet_rowIdsFolder",
-			locator1 = "Checkbox#CHECKBOX_NAME");
+		task ("And selects all documents") {
+			PortletEntry.selectAll();
 
-		Click.javaScriptClick(locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_COLLECT_BUTTON");
+			DigitalSignature.clickCheckboxDM(checkboxDM = "_com_liferay_document_library_web_portlet_DLAdminPortlet_rowIdsFolder");
+		}
 
-		var modalAlert = "You have exceeded the maximum amount of 10 files allowed per envelope.";
+		task ("When the user tries to go to digital signature") {
+			DigitalSignature.goToDigitalSignature();
+		}
 
-		AssertElementPresent(
-			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_MODAL_ALERT",
-			value1 = ${modalAlert});
+		task ("Then expected warning message appears") {
+			DigitalSignature.assertAlertMessage(alertMessage = "You have exceeded the maximum amount of 10 files allowed per envelope.");
 
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
 
-		PortletEntry.selectAll();
+			PortletEntry.selectAll();
 
-		Click(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
+			DigitalSignature.clickCheckboxDM(checkboxDM = "_com_liferay_document_library_web_portlet_DLAdminPortlet_rowIdsFolder");
 
-		VerifyElementPresent(locator1 = "Message#SUCCESS");
+			DigitalSignature.recycleMessageSucess();
+		}
```

Comment:
As I sad above you can remove these delete steps due to they will be added on teardown